Makefile

```diff
@@ -308,6 +308,12 @@ deploy_day2_cloud_nodes_with_install:
 deploy_day2_ocp_nodes_with_install:
 	skipper make $(SKIPPER_PARAMS) _deploy_nodes NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) NAMESPACE=$(NAMESPACE) $(SKIPPER_PARAMS) ADDITIONAL_PARAMS="'-in --day2-ocp-cluster'" DEPLOY_TARGET=ocp
 
+deploy_static_network_config_day2_nodes:
+	skipper make $(SKIPPER_PARAMS) _deploy_nodes NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) NAMESPACE=$(NAMESPACE) $(SKIPPER_PARAMS) ADDITIONAL_PARAMS="'--day2-cloud-cluster --with-static-network-config'"
+
+deploy_static_network_config_day2_nodes_with_install:
+	skipper make $(SKIPPER_PARAMS) _deploy_nodes NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) NAMESPACE=$(NAMESPACE) $(SKIPPER_PARAMS) ADDITIONAL_PARAMS="'-in --day2-cloud-cluster --with-static-network-config'"
+
```

Comment:
Can we use a common target with varying `ADDITIONAL_PARAMS`? It becomes too long, and any new parameter will require changing all the individual targets.

Reply:
i thought about it, but that means that someone new will have to know which parameters to use and one. This way Makefile may be long, but any user can find out how to run a scenario without trying to understand all the parameters need to be passed